scripts/update_docs_version_selector.py

```
@@ -0,0 +1,277 @@
+
+def _is_missing_gh_pages_ref(message: str) -> bool:
+    lowered = message.lower()
+    return (
+        "couldn't find remote ref" in lowered
+        or "not found in upstream origin" in lowered
+    )
+
+
+def fetch_gh_pages(repo_root: Path) -> bool:
+    result = subprocess.run(
+        ["git", "fetch", "origin", "gh-pages"],
```

Ruff (0.15.20)

[error] Starting a process with a partial executable path

(S607)